tests/wps_restapi/test_jobs.py

```diff
@@ -1579,6 +1586,26 @@ def test_jobs_inputs_outputs_validations(self):
         with self.assertRaises(colander.Invalid):
             sd.Execute().deserialize({"outputs": {"random": {"transmissionMode": "bad"}}})
 
+    def test_job_transforms(self):
+        # First try transformation engine
+        test_transformations()
```

Comment:
I don't think we need to test this again here since it should already be done in [tests/transform/test_tranform.py](https://github.com/crim-ca/weaver/pull/548/files#diff-5d661929ab49da9c95d3dcdb9056cdac29f1e3762900a841183ff36c50ebfd8f)